### ISSUE TITLE
Overclocking

### ISSUE THREAD

**Issue by iberras** [JohnCoates/Aerial#335]
There seems to be an issue that the CPU usage peaks. When the Aerial screensaver is running, the fans start to work and they don't stop till the screen saver stops. This was not happening before. I did a clean restart because I thought this was another problem, but it persists.

I have macOS Sierra 10.12.3, retina 13 early 2015.

If you need any more information, please let me know.

**Comment by alistairmcmillan** [comment]
Which version of Aerial are you running? Are you running the latest version from [here](https://github.com/JohnCoates/Aerial/releases/tag/v1.2)? If you have any other versions installed, please remove them.

Do you have the videos cached?

**Comment by iberras** [comment]
The videos are cached. And I have the last version.

**Comment by iberras** [comment]
Hello. The bug persists. The videos are cached. Im running the latest version of macOS in a MacBook Pro retina 2015. The fans start working really hard every time the screen saver is one.

Hope you can find a solution!

**Comment by alistairmcmillan** [comment]
I have the same MacBook. I tried leaving the screensaver running for hours last night. The fans never came on once.

There is a missing piece to this puzzle. Do you have external monitors connected? If you try to run the videos in `/Users/YOURUSERNAME/Library/Caches/Aerial` full screen in QuickTime do the fans come on?

**Comment by iberras** [comment]
I do have an external monitor, ASUS PB287Q 28. I testes with quicktime both videos in the macbook and the external monitor and the fans start lightly to run. I should try a more through test.

Maybe the previous videos stay in a cache after the others played and the memory starts to fill up. I don’t know. Or maybe the MacBook cannot handle the external monitor with the two video.

**Comment by alistairmcmillan** [comment]
The videos definitely don't stay in memory. I tested that recently for another issue. :)

Try disconnecting the external monitor and running the screensaver. See if the fans come on then.

**Comment by iberras** [comment]
The tests I have done, the fan doesn’t run a lot, but they start running. I will test again later and let you know the results.

**Comment by noahsilas** [comment]
I also started encountering this, but only after upgrading MacOS from 10.11 to 10.12.

**Comment by iberras** [comment]
I am in 10.12. I Hope you can find the issue. Though beacuse the videos are so great, that i didnt care that the fans peak!

**Comment by noahsilas** [comment]
After unchecking "Play Different Aerial On Each Display" option, the problem subsided for me. But now I only get to see one beautiful video at a time. (It's a hard life 😜 )

**Comment by iberras** [comment]
I have the same video on both monitors(always have), and the fans work like hell...

**Comment by alistairmcmillan** [comment]
@iberras I ask before but you never answered. If you run the screensaver without your external monitor plugged in do the fans still come on?

**Comment by iberras** [comment]
Yes, absolutly. I have to note that the last Mac update is very unstable in regards of the vídeo driver.

**Comment by glouel** [comment]
Could you check with V1.4 maybe ? Just so you know, another thing that may slow down video playback is iCloud Photo syncing, see the troubleshooting paragraph at the bottom of the project readme, we had someone report that earlier.
